portal-backend/depmap/discourse/client.py

```diff
@@ -1,59 +1,130 @@
 import requests
+from requests.adapters import HTTPAdapter, Retry
 from urllib.parse import urljoin
 from .utils import reformat_date
+from flask import current_app
+import os
+import json
+from sqlitedict import SqliteDict
 
 
 class DiscourseClient:
-    def __init__(self, api_key: str, base_url: str):
+    """The client has two modes:
+    1. In refresh mode, data is fetched from Discourse API and stored in a DB cache
```

Comment:
Add a note that the "reload" parameter is controlling which of these two is in play?